### PR TITLE
Add "@jetmet" to Muon datasets in production configuration

### PR DESCRIPTION
**Pull request by LinaresToine**
As requested by ORM @francescobrivio, the PR to include @jetmet in Muon datasets in the production node.

### CODE REVIEW

**Comment by francescobrivio** [comment]
@LinaresToine thanks!
But I think the `etc/ReplayOfflineConfiguration.py` file that you are modifying was not in sync with `master`?

**Comment by LinaresToine** [comment]
Apparently the replay config file I added was synced with the one at the master branch. I took it from the replay PR.

**Comment by LinaresToine** [comment]
I dont understand, it should be comparing with the dmwm master branch anyways, no?

**Comment by LinaresToine** [comment]
Ok @francescobrivio sorry for that. The files should be good now.

**Comment by LinaresToine** [comment]
test syntax please

File: etc/ReplayOfflineConfiguration.py [hunk 1]
```diff
@@ -1160,6 +1160,9 @@
 ignoreStream(tier0Config, "streamDQMRates")
 ignoreStream(tier0Config, "DQMPPSRandom")
 
+# Run only on Muon0 and Muon1 streams
+#specifyStreams(tier0Config, ["PhysicsMuon0","PhysicsMuon1"])
```
Comment (francescobrivio) [inline etc/ReplayOfflineConfiguration.py:1164]: We can provably remove these commented lines. They will be changed in any case in future replays.

**Comment by francescobrivio** [comment]
Thanks @LinaresToine! Looks good!

@germanfgv do yuo prefer to deploy this in production after the JointOps on Monday?

**Pull request merged**